compiler/rustc_trait_selection/src/solve/search_graph/cache.rs

```diff
@@ -95,6 +95,7 @@ impl<'tcx> ProvisionalCache<'tcx> {
     }
 
     pub(super) fn provisional_result(&self, entry_index: EntryIndex) -> QueryResult<'tcx> {
+        // FIXME: Responses should probably be `Copy` as well
```

Comment:
ah yeah, probably want to also intern `ExternalConstraints` or something :thinking:

Reply:
Yeah i'm gonna do that in a follow up, then we can remove the other clones :D